packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -70,6 +86,28 @@ export const Stackbox = () => {
     setEndDateTime(new Date(endDateByFrequency[frequency]));
   }, [frequency]);
 
+  const formattedBalance = (balanceData: NonNullable<typeof balance>) => {
+    const SIGNIFICANT_DIGITS = 5;
+
+    /* parseFloat as some number size limitations so this is a temporary solution */
```

Comment:
```suggestion
    /* parseFloat has some number size limitations so this is a temporary solution */
```

what are the limitations? i
s the fix so hard or so necessary that we have to keep a comment?

Reply:
Description is clearer now.